releasenotes/notes/57827.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+
+# issue is a list of GitHub issues resolved in this note.
+# If issue is not in the current repo, specify its full URL instead.
+issue:
+- 57827
+
+releaseNotes:
+  - |
+    **Added** support for a user defined environment variable to override iptables backend detection and use a specific binary.
```

Comment:
I think including the env variable would help users find the right config option more easily.

Reply:
Most users should never configure this. Then again, most will not read the release notes so maybe this isn't a compelling argument against adjust the note.